libbeat/processors/dns/config.go

```diff
@@ -84,9 +84,12 @@ type CacheConfig struct {
 // CacheSettings define the caching behavior for an individual cache.
 type CacheSettings struct {
 	// TTL value for items in cache. Not used for success because we use TTL
-	// from the DNS record.
+	// from the DNS record or the minimum configured TTL.
```

Comment:
Is the comment inaccurate? There's a separate field for MinTTL.


```suggestion
	// from the DNS record.
```